File: csharp/ex049.cs
// Crie um programa que leia 6 números inteiros e no final mostre quantos deles
// são pares e quantos são ímpares.

class Ex049
{
  public int counter = 0;
  public int soma = 0;

  public int odd = 0;
  public int even = 0;

  public void FirstCountdown()
  {
    while (counter <= 6)
    {
      Console.WriteLine("Digite um numero: ");
      int inputNumero = Convert.ToInt32(Console.ReadLine());

      if (inputNumero % 2 == 0)
      {
        even += 1;
      }
      else
      {
        odd += 1;
      }


      soma += inputNumero;
    }
    Console.Write($"Pares : {even} , Impares: {odd}");
  }
}

File: python/ex049.py
# Crie um programa que leia 6 números inteiros e no final mostre quantos deles
# são pares e quantos são ímpares.


counter = 0
odd = 0
even = 0

while counter <= 5:
    inputNumber = int(input("Digite um numero inteiro: "))
    if(inputNumber % 2 == 0):
        odd+=1
    else:
        even+=1
    counter+=1
print(f"Numeros pares: {odd} \n Numeros Impares: {even}")